src/app/view-balance/view-balance.component.html

```diff
@@ -0,0 +1,16 @@
+<mat-card class="full-card">
+  <mat-card-title>View Balance</mat-card-title>
+  <form [formGroup]="balForm">
+    <div class="form-group" id="div-tx-form">
+      <mat-form-field class="full-width">
+        <input matInput placeholder="Private Key" formControlName="privateKey">
+      </mat-form-field>
+      <mat-form-field class="full-width">
+        <input matInput placeholder="Address" formControlName="address">
+      </mat-form-field>
+      <div *ngIf="balance">
+        Your current balance is: {{balance}}
+      </div>
```

Comment:
Should we link to explorer here? Something like "See Explorer for a detailed transaction history." with a link to their address.

Reply:
<img width="1356" alt="screenshot 2018-06-16 11 35 02" src="https://user-images.githubusercontent.com/27381/41501444-5dc836e4-7159-11e8-94e0-4fa79117daad.png">